CONTRIBUTING.md

```diff
@@ -120,14 +122,363 @@ rework your branch. You should rework any relevant information from the GitHub
+  $ cd community
+  ```
+
+  > **Note:** The previous Git command:
+  >
+  > - Creates a *local copy* of the upstream repository and switches to the
+  >   `master` *branch*.
+  > - Creates an "alias" (name) for the upstream repository URL called `origin`.
+
+  There are now *three* copies of the repository (all *exactly* the same):
+
+  - The original upstream one (https://github.com/kata-containers/community)
+
+    You do not (and cannot) make changes to this repository. PRs are a way of
+    allowing the project developers and administrators to review your work and
+    decide if they want to add your work to the upstream branch.
+
+    > **Note:** Don't worry - even if you make a mistake,
+    > you *cannot* do any damage to this repository.
+
+  - Your fork (`https://github.com/${your-github-username}/community`) on GitHub
+
+    This is where you send or upload (`git push`) your PR branches.
+
+  - Your *local copy* of the upstream repository
+    (https://github.com/kata-containers/community)
+
+    This is where you actually make changes to your private copy of the
+    repository.
+
+- [Setup a "git remote"](https://help.github.com/articles/configuring-a-remote-for-a-fork):
+
+  "What happened to my fork?". The following demonstrates where your fork is
+  and how you use it.
+
+  Recall that your fork on the GitHub server is an *exact copy* of the
+  upstream repository that also lives on the GitHub server. That might not be
+  true moving forward because PRs are constantly merged into the upstream
+  repository. This means your fork will be out of date because is will not
+  contain the latest changes in the upstream repository. The same
+  problem also affects your local copy of the upstream repository.
+
+  The way you keep your local copy up to date is to run `git pull`
+  periodically. This command downloads all the latest changes from "another
+  repository" into your local repository.
+
+  You might ask how Git knows which repository to pull changes from. Simply,
+  it defaults to pull from the repository you `git clone`'d in previous steps.
+  In Git terminology that default repository is given two names:
+
+  - The URL for the default upstream repository is called the `origin`.
+
+  - The local branch that Git puts the copy of the upstream repository into is
+    called `master`.
+
+  If you run `git push`, Git will try to upload any of your local changes
+  into the default repository (i.e. the `origin`). This will not work because
+  the `origin` is the upstream repository that you do not have permission to
+  change.
+
+  You need to tell Git that your changes should be uploaded to your fork on
+  the GitHub server when you `git push` This is done by adding what is called
+  a "remote":
+
+  ```sh
+  $ user="your-github-username"
+  $ git remote add github https://github.com/${user}/community
+  ```
+
+  > **Note:** The previous command adds an alias (name) for the URL of the
```

Comment:
Truncated sentence needs to be completed.

Reply:
Fixed.